regexes/device/mobiles.yml

```diff
@@ -3882,6 +3926,8 @@ LG:
       model: '$1'
     - regex: '(VX[0-9]+)'
       model: '$1'
+    - regex: '(Nexsus 5)'
```

Comment:
Isn't that the same as Google's `Nexus 5` (which was produced by LG)

Reply:
I'm not really sure it's Google.
Nexsus 5 China DIY device
Nexus 5  Google+LG

Reply:
Ah so it's a copy of the Nexus 5?